source/includes/audit-log/_introduction.md

```diff
@@ -93,6 +93,7 @@ Unless otherwise specified, audit log API methods generally conform to the follo
 
 | Date                          | Description                                                                                                                       |
 |-------------------------------| --------------------------------------------------------------------------------------------------------------------------------- |
-| Nov 6, 2023 | Update Throttling section to Rate Limiting and updated approach to rate limiting.
-| Oct 26, 2023 | Update Pagination to reflect new approach to returning `pit_ids`.
+| Nov 7, 2023 | Removed On-Behalf-Of header from the [events object](#events) sample payload 
+| Nov 6, 2023 | Updated Throttling section to Rate limiting and updated approach to rate limiting.
+| Oct 26, 2023 | Updated Pagination to reflect new approach to returning `pit_ids`.
```

Comment:
Normally I wouldn't like someone updating previous items in a changelog. But this is such a minimal change (Update -> Updated) and on top of that they are your own changelog entries so it doesn't matter anyways :)